views/about.ejs

```diff
@@ -1,123 +1,144 @@
-    <br>
-    <br>
-    
-   
+  </div>
+  <div class="row justify-content-center">
+    <div class="col-md-10 col-sm-12 col-12">
+      <h3 class="marb-20">LADC Life</h3>
+      <p>
+        The Signet is open from 6 a.m. until midnight during the eight months
+        that the College is in session. Members can read, relax, play the piano,
+        and entertain each other and their guests.
+      </p>
+    </div>
+  </div>
 
-</div>
-</div>
+  <div class="row justify-content-center mart-40">
+    <div class="col-md-9 col-sm-10 col-10 border border-secondary"></div>
+  </div>
 
+  <div class="row justify-content-center mart-40">
+    <div class="col-12 col-md-10 col-sm-12">
+      <h3>FIND OUT MORE</h3>
+      <p>
+        Below, you can find a variety of resources to learn more about the
+        people and activities of the Signet Society.
+      </p>
+    </div>
+  </div>
 
-<div class="row justify-content-center">
-    <div class="col-10 col-md-3 col-sm-6 mb-2 col-lg-2 mb-5">
-        <div class="card">
-            <img src="https://scontent.fluh1-1.fna.fbcdn.net/v/t31.0-8/29060567_960144107478014_7890786354507030292_o.jpg?_nc_cat=102&ccb=2&_nc_sid=e3f864&_nc_ohc=BFpkY9kbKvMAX_bj3PA&_nc_ht=scontent.fluh1-1.fna&oh=747c1d86063fc02d731adf8b47b90669&oe=6031C95A" class="card-img-top cimg" alt="...">
-            <div class="card-body">
-              <h5 class="card-title">LADC ALUMNI</h5>
-              <p class="card-text">Some quick example text to build on the card title and.</p>
-              <div style="text-align:center;"><a href="#" class="btn btn-warning">LEARN MORE</a></div>
-            </div>
+  <div class="row justify-content-center mart-40">
+    <div class="col-10 col-md-3 col-sm-6 mb-2 col-lg-2 marb-40">
+      <div class="card">
+        <img
+          src="https://scontent.fluh1-1.fna.fbcdn.net/v/t31.0-8/29060567_960144107478014_7890786354507030292_o.jpg?_nc_cat=102&ccb=2&_nc_sid=e3f864&_nc_ohc=BFpkY9kbKvMAX_bj3PA&_nc_ht=scontent.fluh1-1.fna&oh=747c1d86063fc02d731adf8b47b90669&oe=6031C95A"
+          class="card-img-top cimg"
+          alt="..."
+        />
+        <div class="card-body">
+          <h5 class="card-title">LADC ALUMNI</h5>
+          <p class="card-text">
+            Some quick example text to build on the card title and.
+          </p>
+          <div style="text-align: center">
+            <a href="#" class="btn btn-warning">LEARN MORE</a>
           </div>
-</div>
-<div class="col-10 col-md-3 col-sm-6 mb-2 col-lg-2 mb-5">
-    <div class="card">
-        <img src="https://scontent.fluh1-1.fna.fbcdn.net/v/t1.0-9/117385989_1676103219215429_6284840838078341627_o.jpg?_nc_cat=106&ccb=2&_nc_sid=8bfeb9&_nc_ohc=yOOdwACwpFcAX9Sb2IE&_nc_ht=scontent.fluh1-1.fna&oh=9b0e7ab4030ae5f3b617c33aae5a515e&oe=60300E66" class="card-img-top cimg" alt="...">
+        </div>
+      </div>
+    </div>
+    <div class="col-10 col-md-3 col-sm-6 mb-2 col-lg-2 marb-40">
+      <div class="card">
```

Comment:
height of the card is varying to the content size ,that makes length of each card different . try using text-wrap bootstrap class with some other variations to resolve this

Reply:
Done Sir, I have made height responsive to the content and button will align itself to the bottom of the card.